source/Projectile.cpp

```diff
@@ -417,12 +419,12 @@ void Projectile::CheckLock(const Ship &target)
 			double rangeFraction = min(1., distance / jammingRange);
 			opticalJamming = (1. - rangeFraction) * opticalJamming;
 		}
-		double weight = target.Mass() * target.Mass();
-		double probability = weapon->OpticalTracking() * weight / (150000. + weight) / (1. + opticalJamming);
+		double weight = (target.Mass() * target.Mass()) / 1000000.;
+		double probability = (weapon->OpticalTracking() * weight / 1. + weight) / (1. + opticalJamming);
```

Comment:
What is this division by 1 for?

Reply:
Oops, that should be weight / (1. + weight)